Example/ViewController.swift

```diff
@@ -303,6 +304,18 @@ class ViewController: UIViewController {
 }
 
 extension ViewController: CropViewControllerDelegate {
+    func cropViewControllerDidEnableUndo(_ enable: Bool) {
+        
+    }
+    
+    func cropViewControllerDidEnableRedo(_ enable: Bool) {
+        
+    }
+    
+    func cropViewControllerDidEnableReset(_ enable: Bool) {
+        
+    }
+    
     func cropViewControllerDidCancel(_ cropViewController: CropViewController, original: UIImage) {
```

Comment:
These three implementation can be removed once adding default implementation in CropViewControllerDelegate